src/v2/Apps/Artwork/Components/ArtworkSidebar/index.tsx

```diff
@@ -53,12 +54,21 @@ export const ArtworkSidebar: React.FC<ArtworkSidebarProps> = ({
             />
           </Join>
 
-          {artwork.sale && !artwork.sale?.is_closed && (
-            <>
-              <Spacer mt={2} />
-              <AuctionTimerFragmentContainer sale={artwork.sale} />
-            </>
-          )}
+          {artwork.saleArtwork &&
+            !artwork.saleArtwork.endedAt &&
+            artwork.sale &&
+            !artwork.sale.is_closed &&
+            (artwork.sale?.cascadingEndTimeInterval ? (
```

Comment:
It might be helpful to put all of these logical bits behind a self-describing variable name, eg 
```tsx
const isClosingSoon = foo && !bar && baz.foo && bam 
...

{isClosingSoon && <SomeComponent />
```

Then future self will know what this means :)

Reply:
I put it in a helper because this logic is also used in other files and yes, much better!

Reply:
since we don't have other helpers, we should probably just name the file `lotIsClosed`

Reply:
ok!